src/lib/components/multiDropzone/types.ts

```diff
@@ -1,17 +1,30 @@
 import { Accept } from "react-dropzone";
 
-export type UploadStatus = 'UPLOADING' | 'COMPLETE' | 'ERROR';
+export enum FileMimeTypes {
+  bmp = "image/bmp",
+  doc = "application/msword",
+  docx = "application/vnd.openxmlformats-officedocument.wordprocessingml.document",
+  heic = "image/heic",
+  jpeg = "image/jpeg",
+  jpg = "image/jpg",
+  pdf = "application/pdf",
+  png = "image/png",
+  tif = "image/tiff",
+  tiff = "image/tiff",
+  webp = "image/webp",
+}
 
-export const DOCUMENT_FILES = ['doc', 'docx', 'pdf'];
-export const IMAGE_FILES = ['heic', 'bmp', 'jpeg', 'jpg', 'png'];
+export type FileType = keyof typeof FileMimeTypes;
```

Comment:
It'd be useful to export this from the lib, wdyt?

Reply:
Agree, I think we should start exporting more type definitions from the DS as they can be handy on implementations

Reply:
Updated here: 
https://github.com/getPopsure/dirty-swan/pull/172/commits/39f7f4a96c14b342205173d6335a3431aafe49c5